pandas/_libs/src/inference.pyx

```diff
@@ -3,7 +3,7 @@ from decimal import Decimal
 cimport util
 cimport cython
 from tslibs.nattype import NaT
-from tslib cimport convert_to_tsobject
+from tslibs.conversion cimport convert_to_tsobject
```

Comment:
need to add ``conversion.pxd`` as a dep for lib.pyx (tslib.pyd should have been there)